10up/ElasticPress
# Add isFirstSearch prop to Instant Results

### Description of the Change





This PR adds a new `isFirstSearch` to keep track of whether we are on the first time searching in instant results. If it is set to true instant results will show a "Loading results" text until we have set the results from Elastic.

Closes https://github.com/10up/ElasticPress/issues/3508

### How to test the Change


1.- With instant results activated, search for a term
2.- You should see the text "Loading results" instead of "0 results" for a brief moment

### Changelog Entry

> Fixed - Display "Loading results" instead of "0 results" on first search.

### Credits

Props @oscarssanchez, @JakePT, @felipeelia 


### Checklist:
